gno.land/pkg/integration/doc.go

```diff
@@ -45,6 +45,12 @@
 //   - It's important to note that the load order is significant when using multiple `loadpkg`
 //     command; packages should be loaded in the order they are dependent upon.
 //
+// 6. `patchpkg`:
+//   - Patches any loaded files by package by replacing all occurrences of the first argument with the second.
+//   - This is mostly used to replace hardcoded addresses from loaded packages.
+//   - NOTE: this command may only be temporary, as it's not best approach to
+//     solve the above problem
+//
```

Comment:
I really think we shouldn't do this. 

We can use govDAO to add users to realms on genesis or these tests. Example here: https://github.com/gnolang/gno/blob/master/examples/gno.land/r/gov/dao/prop1_filetest.gno#L20

Reply:
@ajnavarro

This is so we can change the "admin" address in .gno files for the purpose of testing, it's definitely not the perfect approach, but a good patch regardless (we don't have this functionality yet)

Reply:
Yes, I think `GovDAO` is the best approach, but we probably want to unify this everywhere. In the meantime, I think the patch approach is harmless as it's only available for `txtar` in `gnoland`.